lib/onigumo.ex

```diff
@@ -10,16 +10,16 @@ defmodule Onigumo do
     http = http_client()
 
     load_urls(@input_path)
-    |> Enum.map(&download(&1, http))
+    |> Enum.map(&download(&1, http, @output_path))
   end
 
-  def download(url, http) do
+  def download(url, http, path) do
```

Comment:
Just thinking, wouldn’t it be more intuitive to have the literal input data collected on one side and the behaviors on the other side? Considering we already moved `url` to the beginning, I’d bring `path` there too.

```suggestion
  def download(url, path, http) do
```